tests-integration/tests/integration/cnf/cnf1/bool1.expected-sat.solutions.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "a": {
+      "Int": 0
```

Comment:
Solutions should be returned as Booleans.

Reply:
I take it back, I think Minion also uses 0/1 ints at the moment. Another task that should be captured for the future...